components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/index.html

```diff
@@ -1692,52 +1777,71 @@ <h4 class="modal-title file-dialog-title" id="sampleEvent">Generate sample event
                     "artifacts/AggregateDataIncrementally/AggregateDataIncrementally.siddhi"]
             },
             stream_predefined_annotations: [
-                {
-                    predefined: true,
-                    name: 'Async',
-                    class: 'optional-annotation',
-                    isMandatory: false,
-                    elements: [
-                        {
-                            key: 'buffer.size',
-                            value: '256',
-                            class: 'mandatory-key',
-                            isMandatory: true
-                        },
-                        {
-                            key: 'workers',
-                            value: '2',
-                            class: 'mandatory-key',
-                            isMandatory: true
-                        },
-                        {
-                            key: 'batch.size.max',
-                            value: '5',
-                            class: 'mandatory-key',
-                            isMandatory: true
-                        },
-                    ]
-                }
+            {
+                predefined: true,
+                name: 'Async',
+                class: 'optional-annotation',
+                isMandatory: false,
+                elements: [
+                    {
+                        key: 'buffer.size',
+                        value: '256',
```

Comment:
lets have it as defaultValue

Reply:
Fixed in a046ddb3c214ec608ebdf90386019c42b98c4ddc